hexvexed-0.0.1.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="Hexagon game based  for haiku"
```

Comment:
SUMMARY should not start with the name of the program. You have to chose something like:
```
SUMMARY="An hexagon-based game inspired by ..."
```